scripts/test/Ticket/TicketMasks.t

```diff
@@ -0,0 +1,300 @@
+# --
+# OTOBO is a web-based ticketing system for service organisations.
+# --
+# Copyright (C) 2001-2020 OTRS AG, https://otrs.com/
+# Copyright (C) 2019-2025 Rother OSS GmbH, https://otobo.io/
+# --
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# --
+
+use strict;
+use warnings;
+use utf8;
+
+use Test2::V0;
+use Kernel::System::VariableCheck qw(:all);
+
+# Set up the test driver $Self when we are running as a standalone script.
```

Comment:
I recommend to use `Kernel::System::UnitTest::RegisterOM` and do away with the almost unused `$Self`.